test/cmd/images_tests.sh

```diff
@@ -198,6 +198,8 @@ os::cmd::expect_success 'oc new-project test-cmd-images-2'
 os::cmd::expect_success "oc tag $project/mysql:5.5 newrepo:latest"
 os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamImage'
 os::cmd::expect_success_and_text 'oc get istag/newrepo:latest -o jsonpath={.image.dockerImageReference}' 'mysql@sha256:'
+# tag accross projects without specifying the source's project
+os::cmd::expect_success_and_text "oc tag newrepo:latest '${project}/mysql:tag1'" "Tag cmd-images/mysql:tag1 set to newrepo"
```

Comment:
@juanvallejo this still doesn't check that anything happened -- you want to do:
 - expect that the tag command succeeds
 - look at the is and see the new tag